python/lsst/ts/xml/generate_subsystems_doc.py

```diff
@@ -291,12 +291,19 @@ def generate_subsystems_doc():
                             write_heading(cf, name="Enumerations")
                             enumeration_seen = True
                         states = topic.text.split(",")
+                        print(states)
                         state_name = states[0].split("_")[0].lstrip()
                         cf.write(f":{state_name}:\n")
                         for state in states:
+                            state = state.split("_")[1:]
+                            state = "".join(state)
+                            state = state.lstrip()
+                            state = state.split("=")
+                            state = "".join(state[0])
```

Comment:
I am puzzled by these two lines. It looks like you split at an equals and then feed the first item to `join`. Splitting at an equals makes sense to me, but feeding a single string to `join` does not. I am pretty sure that has to be a no-op. (I'm a bit surprised it even works, but strings are iterable).

Also please consider rstrip after the split, because there may be spaces before the `=`.